.github/workflows/release.yaml

```
@@ -25,6 +29,7 @@ jobs:
       - run: npm ci
       - run: npm run build
         env:
+          MV: ${{ matrix.MV }}
           GOOGLE_APP_ID: ${{ secrets.GOOGLE_APP_ID }}
           GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
           GOOGLE_OAUTH_CLIENT_ID: ${{ secrets.GOOGLE_OAUTH_CLIENT_ID }}
```

> * Remove `GOOGLE_*` env variables

Can you expand on this? What's the expected outcome at this point? Is this to trigger some extra error because we'll eventually want to remove these?


If you want this, I think I can drop them with something like this, but I have to test it elsewhere first

```suggestion
          GOOGLE_OAUTH_CLIENT_ID: ${{ matrix.MV == 2 && secrets.GOOGLE_OAUTH_CLIENT_ID }}
```

---

> Can you expand on this? What's the expected outcome at this point?

You can double-check, but I don't think we use those `GOOGLE_` variables in the project anymore now that Google is a normal PKCE integration

---

One is used here, can this be dropped from MV2 as well?

https://github.com/pixiebrix/pixiebrix-extension/blob/7cea09e2bd3c766703c8592e10d0b03fa1aa5cba/webpack.config.js#L239-L246

---

- I opened https://github.com/pixiebrix/pixiebrix-extension/pull/7188

---

I think so - @BLoe could you confirm? I also tagged you on the other PR

---

Yeah, we can definitely drop these as far as I know